airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/http_requester.py

```diff
@@ -553,3 +485,204 @@ def _try_get_error(value: Any) -> Any:
             return str(error) if error else None
         except requests.exceptions.JSONDecodeError:
             return None
+
+
+@dataclass
+class LowCodeHttpRequester(HttpRequester):
+    """
+    Low code default implementation of a Requester
+
+    Attributes:
+        url_base (Union[InterpolatedString, str]): Base url to send requests to
+        path (Union[InterpolatedString, str]): Path to send requests to
+        config (Config): The user-provided configuration as specified by the source's spec
+        request_options_provider (Optional[InterpolatedRequestOptionsProvider]): request option provider defining the options to set on outgoing requests
+    """
+
+    url_base: Union[InterpolatedString, str]
+    path: Union[InterpolatedString, str]
+    config: Config
+    parameters: Mapping[str, Any]
```

Comment:
Replaced `InitVar[Mapping[str, Any]]` with `Mapping[str, Any]`, 
cause we explicitly initialize the parameters through the `__init__` method, and in the end, we have the `_parameters` field as the output in both cases. 
If we don't remove the `InitVar`, then a problem arises when `__post_init__` must take `parameters` as an argument, but we have already passed it to `__init__`, so when calling the `__post_init__` method, we get the error `__post_init__() missing 1 required positional argument: 'parameters'`